README.md

```diff
@@ -1 +1,189 @@
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello_Microverse! <a name="about-project"></a>
+
+
+
+**[Hello_Microverse!]** is the first GitHub repository project created by me, Mateo Burneo in Microverse
+
+## 🛠 Built With <a name="built-with"></a>
+
+  - HTML
+  - CSS
+
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+    <li><a href="https://www.w3.org/Style/CSS/Overview.en.html">CSS</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[present_a_text]**
+- **[add_color_with_CSS]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+-Git <br />
+-Vscode
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+  cd my-folder <br />
+  git clone https://github.com/mateoburvi/hello-microverse.git.
+
+  
+
+### Install
+
+Install this project with:
+
+  cd my-project
+  gem install
+
+Note: this proyect does not require an instalation
+
+### Usage
+
+This project has no practical Usage
+
+### Run tests
+
+You can run Webhint or Styleint in the project to check for linter errors using the following comands.
+
+Webhint:
+
+```
+npx hint .
+```
+
+Styleint:
+
+```
+npx stylelint "**/*.{css,scss}"
+```
+
+
+### Deployment
+
+You can deploy this project using:
+
+-Google Chrome <br />
+-Mozilla Firefox <br />
+-Opera <br/>
+-IE
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Mateoburvi**
+
+- GitHub: [@mateoburvi](https://github.com/mateoburvi)
+- Twitter: [@mateoburvi](https://twitter.com/mateoburvi)
+- LinkedIn: [mateoburvi](https://www.linkedin.com/in/mateoburvi/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- This proyect won't have future features as it is just a test in GitHub
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can give me a star!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse and all the people in the program 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Great work creating the LICENSE.md file and defining the LICENSE for this project. However, I noticed that the file wasn't linked in your README file, so whenever I click this link, it displays a 404 error page. I would advise that you properly link the `LICENSE.md` file by using this link `./LICENSE.md` instead of `./LICENSE`. 👍 😎